launcher/et

```diff
@@ -32,7 +32,7 @@ do
             shift
             ;;
         -h|--help)
-            printf "et (options) [user@]hostname[:port]
+            printf "et (options) [user@]hostname
```

Comment:
* It conflicts with IPv6 literals, i.e. `mistertea@::1`
* There is code to parse out `:host` but the resulting variable isn't actually used anywhere.
* There's already a `-p` option to provide the info, the same way SSH does it.